californium-core/src/main/java/org/eclipse/californium/core/network/EndpointManager.java

```diff
@@ -330,16 +218,24 @@ public synchronized void setDefaultSecureEndpoint(Endpoint endpoint) {
 
 	// Needed for JUnit Tests to remove state for deduplication
 	/**
-	 * Clear the state for deduplication in both default endpoints.
+	 * Clear the state for deduplication in all default endpoints.
 	 */
 	public static void clear() {
 		EndpointManager it = getEndpointManager();
-		if (it.default_endpoint != null)
-			it.default_endpoint.clear();
-		if (it.default_secure_endpoint != null)
-			it.default_secure_endpoint.clear();
-		if (it.default_tcp_endpoint != null)
-			it.default_tcp_endpoint.clear();
+		for (Endpoint endpoint : it.endpoints.values()) {
+			endpoint.clear();
+		}
+	}
+
+	/**
+	 * Clear all endpoints. Stop them.
+	 */
+	public static void clearEndpoints() {
```

Comment:
The naming of this method is a little confusing. If I am not mistaken, this method is about *removing*/*destroying* all default endpoints, not about clearing their internal state, right?